copilot/environments/test/manifest.yml

```diff
@@ -7,9 +7,17 @@ name: test
 type: Environment
 
 # Import your own VPC and subnets or configure how they should be created.
-# network:
-#   vpc:
-#     id:
+# Run this in uat/production only - in the test environments, these should be ad-hoc per deployment
```

Comment:
I wonder if it would be worth removing the uat from the choice at the top, but putting a comment about this up there, although to be honest it'll be relatively obvious, so probably not worth it.  (Self-resolving comment)